govcd/sample_govcd_test_config.json

```diff
@@ -33,6 +33,10 @@
 		"externalNetmask": "255.255.224.0",
 		"internalIp": "192.168.1.10",
 		"internalNetmask": "255.255.255.0",
+		"externalNetwork": "myexternalnet",
+		"externalNetworkPortGroup": "VMNetwork",
+    "externalNetworkPortGroupType": "NETWORK",
```

Comment:
This looks like an additional parameter compared to the GUI. Why do we need it?

Please also add a comment for the users like you did in the YAML sample below.

Reply:
Cause we can use Networks or distributed switches when create external network. It's in GUI but not explicitly showed. You just choose any of them. And you can have network and dswitch with same name.

Reply:
Thanks for adding comments!